admin/jsonConfig.json

```diff
@@ -21,7 +25,7 @@
                             "value": "solcast"
                         }
                     ],
-                    "default": "forecastsolar",
+                    "default": "spa",
```

Comment:
Why changed default?

Reply:
Sorry was to fasten my development. Will revert in another commit.